Description: Permit warnings from compiler
 Remove the -Werror declaration as one interface to event stream
 now croaks.
Author: Dirk Eddelbuettel <edd@debian.org>
Last-Update: 2021-11-30

--- aws-sdk-cpp-only-s3-1.8.117.orig/cmake/compiler_settings.cmake
+++ aws-sdk-cpp-only-s3-1.8.117/cmake/compiler_settings.cmake
@@ -53,7 +53,7 @@ macro(set_gcc_flags)
 endmacro()
 
 macro(set_gcc_warnings)
-    list(APPEND AWS_COMPILER_WARNINGS "-Wall" "-Werror" "-pedantic" "-Wextra")
+    list(APPEND AWS_COMPILER_WARNINGS "-Wall" "-pedantic" "-Wextra")
     if(COMPILER_CLANG)
         if(PLATFORM_ANDROID)
             # when using clang with libc and API lower than 21 we need to include Android support headers and ignore the gnu-include-next warning.
